lib/pages/card_page_juan_pablo.dart

```diff
@@ -81,18 +78,11 @@ class CardPageJP extends StatelessWidget {
               this.mail,
             ),
             SizedBox(
-              height: Constants.cardPaddingExtraSmall,
-            ),
-            _createSecondaryTexts(
-              Icons.location_on_outlined,
-              this.address,
-            ),
-            SizedBox(
-              height: Constants.cardPaddingExtraSmall,
+              height: Constants.cardPaddingSmall,
             ),
             _createSecondaryTexts(
               Icons.phone_android,
-              this.phone,
+              this.linkedIn,
             ),
             SizedBox(
               height: Constants.cardPaddingMedium,
```

Comment:
On this Card Page you can do one of two things. Abstract the widgets into a new widget that receives a SizedBox value and a widget via parameter, to simplify the structure. Or add a padding to the widgets and by that, be able to remove some of the SizedBoxes